ersilia/core/model.py

```diff
@@ -421,9 +422,10 @@ def update_model_usage_time(self, model_id):
 
     def serve(self):
         self.close()
-        self.session.open(model_id=self.model_id)
+        self.session.open(model_id=self.model_id,track_status=self.track_status)
         self.autoservice.serve()
         self.session.register_service_class(self.autoservice._service_class)
+        self.session.register_tracking_status(self.track_status)
```

Comment:
Same, this is unnecessary since we are already writing `track_status` to session JSON.

Reply:
Right, i have made changes